src/vs/workbench/contrib/debug/browser/repl.ts

```diff
@@ -681,6 +679,10 @@ export class Repl extends FilterViewPane implements IHistoryNavigationWidget {
 			});
 
 		this._register(tree.onDidChangeContentHeight(() => {
+			if (this.findIsOpen && !this.replInput.hasTextFocus()) {
```

Comment:
This is just when the find widget is open, not focused right? So I couldn't, say, set a filter in the tree widget and see my debug output auto-scroll, when I have focus in the editor? That seems like a typical scenario.